lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -110,20 +118,25 @@ function Get-WebContentOnCore {
 function Test-ProjectPackage {
   [cmdletbinding()]
   param ($Path, $Algorithm = 'SHA256', $Hash)
+  if (!$env:Valid_ProjectPackage){
+    Write-Verbose "Testing the $Algorithm hash for $path."
+    $ActualHash = if (Is-FIPS) {
+      (Custom-GetFileHash -Algorithm $Algorithm -Path $Path).Hash.ToLower()
+    } else {
+      (Get-FileHash -Algorithm $Algorithm -Path $Path).Hash.ToLower()
```

Comment:
No, Here `Get-FileHash` is powershell utility.  I have renamed  `Get-FileHash` into `Custom-GetFileHash`. I just make less use of that function due to N+ loop between `disposable` and `use` function.